{"success": true, "data": [{"team": {"id": 296, "name": "uxrb5c"}, "date": "2023-06-09T10:02:39.731444+00:00", "challenge_id": 47, "user": {"id": 443, "name": "faetgWGR"}, "challenge": {"value": 100, "category": "misc", "id": 47, "name": "sanity check"}, "id": 58, "type": "correct"}], "meta": {"count": 1}}
